### PR TITLE
Add child_process plugin helper document

### PR DESCRIPTION
**Pull request by cosmo0920**

### CODE REVIEW

File: docs/v0.14/api-plugin-helper-child_process.txt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Code examples:
+
+    :::ruby
+    # Pass block directly. block is executed in 10 second interval.
+    child_process_execute(:exec_awesome_command, @command, interval: 10, mode: [:read]) {|io|
+      # ...
+    }
+
+    # Pass block with existing method. block is executed one-shot.
+    child_process_execute(:exec_awesome_command, @command, immediate: true, mode: [:read], &method(:run))
```
Comment (repeatedly) [inline docs/v0.14/api-plugin-helper-child_process.txt]: Don't need `repeat: false`?

Comment (cosmo0920) [reply docs/v0.14/api-plugin-helper-child_process.txt]: `child_process_execute` does not have `repeat` keyword arguments.
Instead, it has `immediate: true`.

Comment (repeatedly) [reply docs/v0.14/api-plugin-helper-child_process.txt]: So, https://github.com/fluent/fluentd-docs/pull/309/files#diff-1d772de1a7794f79724dd23e54475765R43 this line is not needed?

Comment (cosmo0920) [reply docs/v0.14/api-plugin-helper-child_process.txt]: Oops.

Comment (cosmo0920) [reply docs/v0.14/api-plugin-helper-child_process.txt]: I removed the needless line.

**Pull request merged**